src/sample-graph-data.ts

```diff
@@ -17,7 +19,13 @@ export const defaultLinks = [
   {source: 21, target: 211},
 ]
 
-function graphNode(id, name, x, y, color) {
+function graphNode(
```

Comment:
A comment should be added here to describe this function, similar to render() and _renderNodes().